e2e/LongHaul/service/IotHub.cs

```diff
@@ -83,6 +83,11 @@ public async Task RunAsync(CancellationToken ct)
             }
         }
 
+        public Task<string> GetEventHubCompatibleConnectionStringAsync(CancellationToken ct)
```

Comment:
I'm starting to wonder if we should have this file, or if other classes should just have access to the service client directly.

I originally did it in the device app so I can hand out an interface to parts of the code that shouldn't need access to all the device client stuff - you wouldn't want it to be able to close the connection, for instance. May be an overkill, though.